sycl/unittests/misc/OsUtils.cpp

```diff
@@ -54,12 +54,9 @@ bool isSameDir(const char* LHS, const char* RHS) {
 class OsUtilsTest : public ::testing::Test {
 };
 
-// This test fails with preview breaking changes enabled.
 // Failure tracker: https://github.com/intel/llvm/issues/19626
-#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
-TEST_F(OsUtilsTest, getCurrentDSODir) {
+/*TEST_F(OsUtilsTest, getCurrentDSODir) {
```

Comment:
I think gtest might support renaming the test to `*_UNSUPPORTED` to exclude it. That way will make sure it still compiles but won't run it.

Reply:
It seems that gtest doesn't have `*_UNSUPPORTED`, so changed to GTEST_SKIP.